holoviews/plotting/bokeh/raster.py

```diff
@@ -96,9 +96,9 @@ def get_data(self, element, ranges, style):
 
         dh, dw = t-b, r-l
         if self.invert_xaxis:
-            l, r = r, l
+            dw = -dw
         if self.invert_yaxis:
-            b, t = t, b
+            dh = -dh
```

Comment:
D'oh! I guess this was always just a bug, since dh,dw have already been computed and thus l,r,b,t will just be ignored?  Damn.

Reply:
I think it mostly works in a lot of scenarios, but not others. Honestly not quite sure.